recipes/marslab/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "marslab" %}
+{% set version = "0.9.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/marslab-{{ version }}.tar.gz
+  sha256: 2b4bd12019b307f22e691f3f3743d17846c7da76d65592a15b5c2f210937c42a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - astropy
+    - clize
+    - cytoolz
+    - dustgoggles
+    - fs
+    - hypothesis
+    - matplotlib-base
+    - more-itertools
+    - numpy
+    - pandas
+    - pathos
+    - pdr
+    - pytest
+    - pytest-cov
+    - python-dateutil
+    - python-Levenshtein
+    - scipy
+    - sympy
+
+test:
+  imports:
+    - marslab
+  commands:
+    # removed pip check due to discrepancies with clize metadata in PyPI vs conda-forge
```

Comment:
Also, I'm unclear as to why `pip check` is causing issues when here and upstream there is no version restrictions on the dependencies.